docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -119,8 +119,16 @@ <h5>
           <tr>
             <td>835</td>
             <td>GE Server gets its URL from the client instead of itself</td>
-            <td>Fixed code so that when you publish DB, GEServer's URL will be used and not the URL which we get from the Publish request message</td>
-          </tr>	  
+            <td>Fixed code so that when you publish DB, GEServer's URL will be used and not the URL which we get from the Publish request message.</td>
+          </tr>        
+          <tr>
+            <td>877</td>
+            <td>Fusion RPM install scripts to only <code>chown</code> the asset root when necessary</td>
+            <td>
+              Recursively <code>chown</code> only when the asset root directory has incorrect ownership on a Fusion master host.
+              This allows upgrading Fusion on machines where changing asset root ownership recursively would be very expensive.
```

Comment:
minor: `very expensive` --> `time expensive` or `time consuming`

Reply:
What is the reason for this proposed change?  What is it trying to address?

Reply:
it was a minor comment: expensive was more a generic term for commodities ...wanted to clarify that we are talking mostly about expensive in term of time (and not $). It is a minor comment that does not hold the PR.